common/scala/src/main/scala/org/apache/openwhisk/core/service/DataManagementService.scala

```diff
@@ -0,0 +1,6 @@
+package org.apache.openwhisk.core.service
+
+case class UpdateDataOnChange(key: String, value: String)
+
+// TODO, all operations for etcd via DataManagementService
```

Comment:
I hope this PR is rebased after this kind of module is landed according to this plan:
https://cwiki.apache.org/confluence/display/OPENWHISK/Component+Design